src/projection-multi/projection-multi-cmake.el

```diff
@@ -141,6 +141,8 @@ When set the generated targets will be prefixed with PROJECT-TYPE."
    (projection--current-project 'no-error)
    `((t ,#'projection-multi-cmake-targets))))
 
+;;;###autoload
+(defvar projection-project-type-cmake)
 ;;;###autoload
 (with-eval-after-load 'projection-types
   (projection-type-append-compile-multi-targets projection-project-type-cmake
```

Comment:
Could you stack these together to make it clearer their needed for each other. Something like:

```suggestion
;;;###autoload
(progn
  (defvar projection-project-type-cmake)
  (with-eval-after-load 'projection-types
    (projection-type-append-compile-multi-targets projection-project-type-cmake
```